From: Keith Packard <keithp@keithp.com>
Date: Mon, 4 Oct 2021 09:47:25 -0700
Subject: Disable creation of Github buttons in sphinx documentation

---
 docs/conf.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/docs/conf.py b/docs/conf.py
index 629757b..b1a82b1 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -97,6 +97,8 @@ html_theme_options = {
     "logo_text_align": "center",
     "github_user": "mu-editor",
     "github_repo": "mu",
+    "github_banner": "false",
+    "github_button": "false",
 }
 
 html_sidebars = {"**": ["about.html", "navigation.html", "searchbox.html"]}
