From: Keith Packard <keithp@keithp.com>
Date: Wed, 4 May 2022 16:17:03 -0700
Subject: Allow later jupyter-client

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 202194d..1d4375e 100644
--- a/setup.py
+++ b/setup.py
@@ -33,7 +33,7 @@ install_requires = [
     # + ';"arm" not in platform_machine and "aarch" not in platform_machine',
     # FIXME: jupyter-client added for Py3.5 compatibility, to be dropped after
     # Mu v1.1 release. So, qtconsole < 5 and jupyter-client < 6.2 (issue #1444)
-    "jupyter-client>=4.1,<6.2",
+    "jupyter-client>=4.1",
     # FIXME: ipykernel max added for macOS 10.13 compatibility, min taken from
     # qtconsole 4.7.7. Full line can be removed after Mu v1.1 release.
     # Dependency mirrored for user venv in mu/wheels/__init__.py
